0_global_functions.R

```diff
@@ -1,3 +1,67 @@
+# Checks whether a variable is a dataframe. Considers also logicals and null values.
+data_check <- function(df){
+  
+  if (is.data.frame(df)){
+    if(nrow(df) > 0){
+      check <- TRUE
+    }else{
+      check <- FALSE
+    }
+  }else{
+    check <- FALSE
+  }
+  
+  return(check)
+  
+}
+
+# Checks whether a value is null or blank
+is_blank_na <- function(x){
+  if(is.na(x) | x == ""){flag = TRUE}else{flag = FALSE}
+  flag
+}
+
+read_file <- function(file_name){
```

Comment:
This should probably get another name or not be used, as it could mask the readr::read_file() function and lead to unexpected behaviour/errors